tests/link/pytorch/test_elemwise.py

```diff
@@ -57,6 +58,72 @@ def test_pytorch_elemwise():
     compare_pytorch_and_py(fg, [[0.9, 0.9]])
 
 
+@pytest.mark.parametrize("axis", [None, 0, 1])
```

Comment:
Can we parametrize these tests with the reduce function? Since they all look the same, we can reduce a bunch of lines. Or at least separate only those that need numerical inputs from those that need boolean (all and any).

Reply:
Also I would like to test axis = (1, 2), and have `a_pt` be a tensor3, so that we cover the case with more than 1 axis, but not all of them.